src/Translation/StatementTranslator.php

```
@@ -46,4 +64,81 @@ private function snakWithSimpleDataValueToDataItem( PropertyValueSnak $snak ): S
+		wfDebug('smw: translate DI: '.get_class($di));
+		if( $di instanceof DIWikiPage ) {
+			return $di->getTitle()->getText()."";
+		} else if ( $di instanceof SMWDITime){
+			wfDebug('time:'.$di->getSerialization());
+			return $di->asDateTime()->format( 'Y-m-d H:i:s' );
+			#return "01.01.1700";
+		}
+		wfdebug('default');
+		return $di."";
+
+	}
```

PHPMD (2.15.0)

[warning] Avoid unused private methods such as 'valueDIToString'. (undefined)

(UnusedPrivateMethod)